data/hai/hai reveal 1 intro.txt

```diff
@@ -11,47 +11,835 @@
+			`	"Be very careful if you do though, they're very likely to shoot at you."`
+				goto questions
+			label help
+			action
+				set "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+			`	She shrugs. "I've heard talk of some things that have been suggested to help, but I don't know myself. I guess you should just do the right thing, whatever that is."`
+			`	She seems to be at a loss for what else to say.`
+				goto questions
+			label end
+			`	"No, thank you for caring," she says. "Whatever you do, just be cautious about it. People are scared, and I have a hunch that when they're under pressure, the Hai are a little stranger than people think they are.`
+			`	"Safe travels, Captain."`
+				decline
+	on decline
+		clear "Hai Reveal Concern Period: Raid on Allhome (local): label help"
+		clear "Hai Reveal Concern Period: Raid on Allhome (local): label unfettered"
+
+mission "Hai Reveal Concern Period: Raid on Allhome (away)"
+	landing
+	invisible
+	source
+		government "Hai"
+		not planet "Allhome"
+		attributes "hai: human presence"
+		not attributes "uninhabited" "retirement"
+	to offer
+		has "event: hr: pirate uptick"
+		has "First Contact: Unfettered: offered"
+		not "Hai Reveal Concern Period: Raid on Allhome (local): offered"
+		random < 20
+	on offer
+		set "hr: allhome raided"
+		conversation
+			`A wall screen situated between two food stands just outside the docking area is playing through a news segment, and your ears suddenly perk up at the word "marauder".`
+			`	"Yesterday, a marauder terrorized the inhabitants of Allhome. They arrived by posing as a trader to investigate rumors currently circulating in human space about our existence. Hai security failed to identify the danger, and the individual snuck his ship in and landed at the spaceport without any opposition. Then his crew hijacked a vessel with some of the crew still on board and made off with it successfully. At this time it is unclear what has happened to those crewhai."`
+			`	At a table nearby, a Hai says, "Oh, that's awful!" and the human next to her responds, "Feels like being back on Solace."`
+			choice
+				`	(Approach the pair.)`
+				`	(Keep walking.)`
+					goto walking
+			
+			`	As you approach the human, a relatively young-looking man, he glances up at you. "Oh hello, just arrived, did you? You didn't happen to see an Argosy out there on your way in?"`
+			`	"Oh shush," says the Hai, "Your uncle will be here when he said he will, no need to bother random folks just going about their business."`
+			`	She looks at you with a wary eye, but then seems to relent. "Though I suppose you did come over here of your own volition, didn't you. Can we help you at all, Captain?"`
+			choice
+				`	"Is this a common problem for Allhome?"`
```

Comment:
Bene-dictator’s reaction to crewhai was my immediate reaction too. I think there should be an option to ask the Hai what the broadcast meant by “crewhai.”

Reply:
In the context of everything else, it's a pretty trivial point though, and you're not next to anyone to ask it at the precise moment that it comes up.
So, I guess I'm asking, how would you suggest tackling it in the context of this mission structure?

Reply:
OK, I think I get it now, I’m just used to the term “crewmembers” which could be applied to Hai just as much as it could be applied to humans. I’m not sure if that can be used instead, but if two people found the same issue, that means it may not just be me who sees that term used more now, whereas if I was used to the term “crewmen” I would probably understand without needing an explanation. Thoughts?